package.json

```diff
@@ -43,5 +43,8 @@
     "@typescript-eslint/parser": "^2.7.0",
     "eslint": "^5.0.0 || ^6.0.0",
     "eslint-plugin-vue": "^5.2.3 || ^6.0.0"
+  },
+  "dependencies": {
+    "vue-eslint-parser": "^7.0.0"
```

Comment:
Here we add vue-eslint-parser as a dependency rather than a peerDependency, otherwise yarn will throw this kind of error
```
Error: A package is trying to access a peer dependency that should be provided by its direct ancestor but isn't
Required package: vue-eslint-parser (via "vue-eslint-parser")
Required by: @vue/eslint-config-typescript
```
And it is arguably right to say it has a direct dependency to it anyways